jekyll/_cci2/server/v4.4/operator/operator-overview.adoc

```diff
@@ -0,0 +1,85 @@
+---
+contentTags:
+  platform:
+    - Server v4.4
+    - Server Admin
+---
+= Operator overview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn about the various tasks and tools involved in administering an installation of CircleCI server.
+:icons: font
+:toc: macro
+:toc-title:
+
+This guide contains information for CircleCI server operators, or those responsible for ensuring CircleCI server is running properly through maintenance and monitoring. Before reading this operator guide, ensure you have read the xref:../overview/circleci-server-overview#[CircleCI server v4.4 overview].
+
+CircleCI server schedules CI/CD jobs using the link:https://www.nomadproject.io/[Nomad] scheduler. The Nomad Server (control plane) runs inside the Kubernetes cluster, while Nomad clients are provisioned outside the cluster. The Nomad clients need access to the Nomad control plane and Kong.
+
+CircleCI server can run Docker jobs on the Nomad clients, and also in dedicated virtual machines for remote docker jobs.
+
+Job artifacts and outputs are sent directly from jobs in Nomad to object storage (S3, GCS, or other supported options).
+
+Audit logs and other items from the application are also stored in object storage, so both the Kubernetes cluster and the Nomad clients need access to object storage.
+
+[#execution-environment]
+== Execution environment
+
+CircleCI server v4.4 uses Nomad as the primary job scheduler. Refer to our xref:introduction-to-nomad-cluster-operation#[Introduction to Nomad Cluster Operation] to learn more about the job scheduler and how to perform basic client and cluster operations.
+
+CircleCI Nomad clients automatically provision compute resources according to the executors configured for each job in a project’s `.circleci/config.yml` file.
+
+[#nomad-clients]
+=== Nomad clients
+Nomad Clients run without storing state, allowing you to increase or decrease the number of containers as needed.
+
+If a job's resource class requires more resources than the Nomad client's instance type has available, it will remain in a pending state. Choosing a smaller instance type for Nomad clients is a way to reduce cost, but limits the Docker resource classes CircleCI can use. Review the xref:../../../configuration-reference#resourceclass[available resource classes] to decide what is best for you. The default instance type will run up to `xlarge` resource classes.
```

Comment:
maybe specify something like "the default instance type as defined in the server-terraform repository" will run up to xlarge, etc.